Add cloud native sustainability website page

leonardpahlke opened the pull request:
ref: https://github.com/cncf/tag-env-sustainability/issues/132

This PR adds a page about the cloud native sustainability week (#95) to https://tag-env-sustainability.cncf.io/about/ 

cc @guidemetothemoon

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+---
+title: CNCF Cloud Native Sustainability Week
+description: Cloud Native Sustainability Week is an event to organize meetings in CNCF community groups around the theme of Cloud Native Sustainability during the second week of October 2023.
```
Suggestion: Cloud Native Sustainability Week is a global event where CNCF community groups organize local meetups around the topic of cloud native sustainability. Cloud Native Sustainability Week will take place during the second week of October 2023.

catblade replied:
Do we want to say community groups or "where the CNCF community organizes local meetups..."

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+slug: cloud-native-sustainability-week
+---
+
+The CNCF Global Week of Cloud Native Sustainability is an event organized by the [Cloud Native Computing Foundation (CNCF)](http://cncf.io) community to address the emerging topic of environmental sustainability in the cloud native industry and open-source space. The event aims to engage with the community and industry, understand the current landscape, and promote collaboration and knowledge sha ...
```
Change to "open source" to comply with naming convention.
Suggestion: "The event aims to engage with the community and IT industry as a whole, get a better understanding of  the current environmental sustainability landscape, and promote collaboration and knowledge sharing on the topic."

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+
+The CNCF Global Week of Cloud Native Sustainability is an event organized by the [Cloud Native Computing Foundation (CNCF)](http://cncf.io) community to address the emerging topic of environmental sustainability in the cloud native industry and open-source space. The event aims to engage with the community and industry, understand the current landscape, and promote collaboration and knowledge sha ...
+
+During the **second week of October 2023 (W41)**, the [broader Cloud Native community](https://community.cncf.io/chapters/) will organize meetups in their cities with a focus on cloud native sustainability. These meetups can be organized by anyone interested in supporting the effort. Additionally, the [CNCF Environmental Sustainability Technical Advisory Group (TAG ENV)](http://github.com/cncf/ta ...
```
- [broader Cloud Native community] -> either "broader cloud native community" to comply with naming convention or "broader CNCF community" or "CNCF community groups around the globe".

- Suggestion to also add encouragement and how to contact us if someone is interested in supporting the effort and hasn't reached out yet, as part of following sentence, for example: "These meetups can be organized by anyone interested in supporting the effort. If you would like to contribute, check out the "Contact and Resources" section below."

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+
+During the **second week of October 2023 (W41)**, the [broader Cloud Native community](https://community.cncf.io/chapters/) will organize meetups in their cities with a focus on cloud native sustainability. These meetups can be organized by anyone interested in supporting the effort. Additionally, the [CNCF Environmental Sustainability Technical Advisory Group (TAG ENV)](http://github.com/cncf/ta ...
+
+1. **Local in-person events**: Meetups organized worldwide during the second week of October 2023 by [CNCF community groups](https://community.cncf.io/chapters/) about cloud native sustainability.
```
- Suggestion: add a sentence above the list saying something like "There will be two types of events organized during the Cloud Native Sustainability Week:"

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+
+## Event Goals
+
+1. **Understand the current [landscape](https://tag-env-sustainability.cncf.io/about/landscape/)**: Engaging with the community and industry to better understand the [landscape](https://tag-env-sustainability.cncf.io/about/landscape/) of cloud native environmental sustainability.
```
Suggestion: "1. **Gain better insights into the current [cloud native environmental sustainability landscape](https://tag-env-sustainability.cncf.io/about/landscape/)**: Engage with the community and IT industry as a whole to develop a deeper understanding of the [cloud native environmental sustainability landscape](https://tag-env-sustainability.cncf.io/about/landscape/). "

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+## Event Goals
+
+1. **Understand the current [landscape](https://tag-env-sustainability.cncf.io/about/landscape/)**: Engaging with the community and industry to better understand the [landscape](https://tag-env-sustainability.cncf.io/about/landscape/) of cloud native environmental sustainability.
+2. **Identify Knowledge Gaps**: Identifying areas of knowledge gaps and further exploration in cloud native environmental sustainability.
```
Suggestion: "Identify knowledge gaps: Recognize domain areas for cloud native environmental sustainability where further exploration and knowledge sharing is needed."

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+
+1. **Understand the current [landscape](https://tag-env-sustainability.cncf.io/about/landscape/)**: Engaging with the community and industry to better understand the [landscape](https://tag-env-sustainability.cncf.io/about/landscape/) of cloud native environmental sustainability.
+2. **Identify Knowledge Gaps**: Identifying areas of knowledge gaps and further exploration in cloud native environmental sustainability.
+3. **Foster project collaboration**: Facilitating collaboration among participants on ongoing projects and tools related to cloud native sustainability.
```
Suggestion: "Foster project collaboration: Encourage participants to actively collaborate on further development and improvement of existing projects and tools that pertain to the field of cloud native sustainability."

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+1. **Understand the current [landscape](https://tag-env-sustainability.cncf.io/about/landscape/)**: Engaging with the community and industry to better understand the [landscape](https://tag-env-sustainability.cncf.io/about/landscape/) of cloud native environmental sustainability.
+2. **Identify Knowledge Gaps**: Identifying areas of knowledge gaps and further exploration in cloud native environmental sustainability.
+3. **Foster project collaboration**: Facilitating collaboration among participants on ongoing projects and tools related to cloud native sustainability.
+4. **Raise awareness**: Increasing awareness and knowledge about cloud native environmental sustainability.
```
Suggestion: "Increase awareness: Enhance awareness and knowledge sharing around the topic of cloud native environmental sustainability."

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+---
```
Thanks for creating this website page, @leonardpahlke, it's taking a really good shape! I have a few suggestions to the content below.

I have an overall suggestion as well: this may be one of our highlight events this year and I think it would be valuable to make it more visible when people access the TAG ENV website. As of now it's located in the "About" section which I don't feel is quite a suitable place for this specific page since it's also related to an event. Can we either make a separate tab in the nav menu of the website, side by side with "About" or put it under "Events" where I feel it would be a more natural place for this kind of content to reside? 😊

leonardpahlke replied:
Agree, let move it!

guidemetothemoon commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+
+## Acknowledgements ✨
+
+The event is organized by volunteers in their spare time. Its truly a community effort without that this would not be possible. Thanks everyone for helping!
```
Suggestion: "The event is organized solely by volunteers who dedicate their spare time, making it a true community effort. Without the collective contributions of everyone involved, this event would not have been possible. Thank you to each and every community member for your support and contributions!"

catblade commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+
+The event is organized by volunteers in their spare time. Its truly a community effort without that this would not be possible. Thanks everyone for helping!
+
+Special thanks to Nancy Chauhan, Adriana Villela, Kristina Devochko, Niki Manoledaki, Mercy Bamiduro, Mehmet, Rodolfo Martínez Vega and Leonard Pahlke to help organise the event.
```
Is this in a specific order?  Or should it be alphabetic?

catblade commented on website/content/about/CloudNativeSustainabilityWeek.md:
```diff
@@ -0,0 +1,80 @@
+
+Special thanks to all the local meetup organizers.
+
+Special thanks to Katie Gamanji, Ricardo Rocha, Wojtek Cichoń, Nikhita Raghunath, Tom Kerkhove, Alena Prokharchyk, Abdel Sghiouar, Huamin Chen, Nate W. and Emily Fox for facilitating, sharing their thoughts and reaching out to people that might be interested organizing a meetup.
```
Same again here.  Specific order?  Or should we do alphabetic?

leonardpahlke replied:
we can make it alphabetic. Good point!

leonardpahlke commented:
Updated the PR, thanks @guidemetothemoon & @catblade! Could you take another look? I moved the page so its now accessible over the menu bar. (maybe we could move the landscape doc too @catblade wdyt?

![Screenshot by Dropbox Capture](https://github.com/cncf/tag-env-sustainability/assets/58054077/0f09879d-50b0-4dd3-9570-df47e183a8e7)

leonardpahlke commented:
Opened this issue btw: https://github.com/cncf/tag-env-sustainability/issues/146

catblade commented:
> Updated the PR, thanks @guidemetothemoon & @catblade! Could you take another look? I moved the page so its now accessible over the menu bar. (maybe we could move the landscape doc too @catblade wdyt?

You mean put it where the bar is at the top?  I think that logical, being that it's one of our primary docs for now.  We may need to restructure as we get more deliverables out.

catblade commented on website/content/cloud-native-sustainability-week/_index.md:
```diff
@@ -0,0 +1,89 @@
+
+There will be two types of events organized during the Cloud Native Sustainability Week:
+1. **Local in-person events**: Meetups organized worldwide during the second week of October 2023 by [CNCF community groups](https://community.cncf.io/chapters/) about cloud native sustainability.
+2. **Virtual Event**: A virtual meetup organized by the CNCF TAG ENV.
```
suggestion:  "Virtual Events" (plural)

catblade replied:
Just realized there is only one event and I read poorly... So ignore this one.

catblade commented on website/content/cloud-native-sustainability-week/_index.md:
```diff
@@ -0,0 +1,89 @@
+*Note that this list is changing, we will publish a complete overview a few weeks for the event! Take a look at this [document](https://docs.google.com/document/d/1s28lqqc3IBAMw4T7n13zfyAuUEEc3I4_xhPxCLA4_dk/edit#heading=h.19phjl5j6fdw) to check the latest status, and persons to contact.*
+
+| **Nr** | **Name** | **Location** | **Date** | **Event Link** |
+|---|---|---|---|---|
```
Can we group these by continent, maybe?  I'd probably do location first, date, event, then name (this is the relevance I do).

leonardpahlke replied:
good idea

catblade commented:
@leonardpahlke  looking really good.  Sorry for the additional items-missed them first round through, but after those are addressed, good to go on my side.

leonardpahlke commented:
Sounds good. Thanks for the reviews! We will update this page over the next months approaching the event. Some visuals would be nice, but we can add those later.

guidemetothemoon commented on website/content/cloud-native-sustainability-week/_index.md:
```diff
@@ -0,0 +1,89 @@
+1. **Gain better insights into the current [cloud native environmental sustainability landscape](https://tag-env-sustainability.cncf.io/about/landscape/)**: Engage with the community and IT industry as a whole to develop a deeper understanding of the [cloud native environmental sustainability landscape](https://tag-env-sustainability.cncf.io/about/landscape/).  
+2. **Identify knowledge gaps**: Recognize domain areas for cloud native environmental sustainability where further exploration and knowledge sharing is needed.
+3. **Foster project collaboration**: Encourage participants to actively collaborate on further development and improvement of existing projects and tools that pertain to the field of cloud native sustainability.
+4. **Raise awareness**: Increase awareness: Enhance awareness and knowledge sharing around the topic of cloud native environmental sustainability.
```
Minor edit: remove "Increase awareness:" after **Raise awareness**:

leonardpahlke commented:
Updated, thanks @guidemetothemoon & @catblade

SamYuan1990 commented on website/content/cloud-native-sustainability-week/_index.md:
```diff
@@ -0,0 +1,90 @@
+| 22 | Taiwan, Taipei | *tbd* | *tbd* | Phil Huang |
+| 23 | USA, Chicago | *tbd* | *tbd* | Josh Gavant |
+| 24 | USA, Oakland | *tbd* | *tbd* | Rynn Mancuso |
+| 25 | USA, Seattle | *tbd* | *tbd* | Yiming Peng|
```
One quick question, if we want to contribute with meet up, how should I update my contact info here?
as information below?
| 26 | China, Beijing | *tbd* | *tbd* | Sam Yuan(github id: SamYuan1990) |

catblade replied:
@SamYuan1990  you'd likely just want to add in a PR with the contact info added.  @leonardpahlke  is this what you imagined?

leonardpahlke replied:
Yes, you can just open a PR and update the details.

leonardpahlke replied:
We will reach out to all in a few weeks and update info's as a batch. But.. Feel free to open PRs at anytime to change the update the records! thanks @SamYuan1990 :)

SamYuan1990 replied:
https://github.com/cncf/tag-env-sustainability/pull/147 here you are :-) please advise.

guidemetothemoon reviewed (approved):
LGTM - great work, Leo! 😊
